src/Microsoft.AspNetCore.Razor.Tools/Roslyn/CompilerServerLogger.cs

```diff
@@ -13,10 +12,10 @@ namespace Microsoft.CodeAnalysis.CommandLine
 {
     /// <summary>
     /// Class for logging information about what happens in the server and client parts of the 
-    /// Roslyn command line compiler and build tasks. Useful for debugging what is going on.
+    /// Razor command line compiler and build tasks. Useful for debugging what is going on.
     /// </summary>
     /// <remarks>
-    /// To use the logging, set the environment variable RoslynCommandLineLogFile to the name
+    /// To use the logging, set the environment variable RAZORBUILDSERVER_LOG to the name
     /// of a file to log to. This file is logged to by both client and server components.
     /// </remarks>
     internal class CompilerServerLogger
```

Comment:
Can you rename and move out of this folder if you are happy with it? Ultimately we want to clean out anything in the Roslyn folder.